crates/pop-cli/src/commands/up/parachain.rs

```diff
@@ -138,6 +145,16 @@ impl ZombienetCommand {
 					}
 				}
 
+				if self.setup_script.is_some() {
+					spinner.set_message("Initializing your setup script...");
+					// sleep for 15 seconds to allow the network to start up
+					sleep(Duration::from_secs(15)).await;
```

Comment:
@evilrobot-01 I am not comfortable with using `sleep` to make sure the network is up. Is there any other way of solving this? `sleep` could vary per machines performance and how complex the network that the user is spinning up e.g. 1 parachain vs 10 parachains